### PR TITLE
1642 psss vis num sentinel sites

### PR DESCRIPTION
**Pull request by IgorNadj**
### Changes:
- Minor: sort permission groups by name (Admin Panel)
- Add latestDataValue data builder
- Add PSSS Tupaia permission group
- Add num sentinel sites dashboard

### Screenshots:
https://github.com/beyondessential/tupaia-backlog/issues/1642#issuecomment-817406499

### CODE REVIEW

File: packages/web-config-server/src/apiV1/dataBuilders/generic/latestData/latestDataValue.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+export const latestDataValue = async ({ models, dataBuilderConfig, query, entity }, aggregator, dhisApi) => {
+  const builder = new LatestDataValueBuilder(
+    models,
+    aggregator,
+    dhisApi,
+    dataBuilderConfig,
+    query,
+    entity,
+    AGGREGATION_TYPES.MOST_RECENT,
```
Comment (billli0) [inline packages/web-config-server/src/apiV1/dataBuilders/generic/latestData/latestDataValue.js:27]:
```suggestion
    aggregator.aggregationTypes. MOST_RECENT,
```

With this we don't need to import aggreegation types from dist.

Comment (billli0) [reply packages/web-config-server/src/apiV1/dataBuilders/generic/latestData/latestDataValue.js:27]: An example: [ValuesPerPeriodByOrgUnit.js L72](https://github.com/beyondessential/tupaia/blob/857481c74d59f537026e59aa8927269a86ee7b35/packages/web-config-server/src/apiV1/dataBuilders/generic/comparison/valuesPerPeriodByOrgUnit.js#L72)

**Review by billli0 — CHANGES_REQUESTED** [review]
Looks good! A minor suggestion to stick with the data builder convention.

File: packages/web-config-server/src/apiV1/dataBuilders/generic/latestData/latestDataValue.js (hunk 1 above)
Comment (IgorNadj) [reply packages/web-config-server/src/apiV1/dataBuilders/generic/latestData/latestDataValue.js]: Updated

File: packages/web-config-server/src/apiV1/dataBuilders/generic/latestData/latestDataValue.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/**
+ * Tupaia Config Server
+ * Copyright (c) 2019 Beyond Essential Systems Pty Ltd
+ */
+
+import { DataBuilder } from '/apiV1/dataBuilders/DataBuilder';
+import { AGGREGATION_TYPES } from '@tupaia/aggregator/dist/aggregationTypes';
```
Comment (billli0) [inline packages/web-config-server/src/apiV1/dataBuilders/generic/latestData/latestDataValue.js:7]:
```suggestion
```

File: packages/web-config-server/src/apiV1/dataBuilders/generic/latestData/latestDataValue.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+export const latestDataValue = async ({ models, dataBuilderConfig, query, entity }, aggregator, dhisApi) => {
+  const builder = new LatestDataValueBuilder(
+    models,
+    aggregator,
+    dhisApi,
+    dataBuilderConfig,
+    query,
+    entity,
+    aggregator.aggregationTypes. MOST_RECENT,
```
Comment (billli0) [inline packages/web-config-server/src/apiV1/dataBuilders/generic/latestData/latestDataValue.js:27]:
```suggestion
    aggregator.aggregationTypes.MOST_RECENT,
```

**Review by billli0 — APPROVED** [review]
Preapprove 👍

**Pull request merged**